localstripe/resources.py

```diff
@@ -738,6 +738,8 @@ def __init__(self, name=None, description=None, email=None,
                 assert type(invoice_settings['default_payment_method']) is str
                 assert (invoice_settings['default_payment_method']
                         .startswith('pm_'))
+            if payment_method is not None:
+                assert type(payment_method) is str
```

Comment:
Can you move this block just before `except AssertionError:`?
We keep the different properties checks ordered as they are passed in `__ini__`.
Thanks!

Reply:
Addressed in a02f078